src/app/chat/component/new-message-window/new-message-window.component.scss

```diff
@@ -25,6 +20,20 @@
   }
 }
 
+@media (max-width: 576px) {
```

Comment:
Please use existing or create new mixin for @media instead of max-width

Reply:
We don't have any mixing for media in prod-branch